src/cases/picoes/add_remove.js

```diff
@@ -0,0 +1,26 @@
+import { World } from "picoes";
+
+function Box(value = 0) {
+  this.value = value;
+}
+
+export default (count) => {
+  let world = new World();
+
+  world.component("a", Box);
```

Comment:
There's no need to register "a" or "b" components as Box, so lines 10-11 can be deleted. Also can delete lines 3-5 since Box would be unused.